src/atomate2/vasp/schemas/phonons.py

```diff
@@ -0,0 +1,474 @@
+    Collection to store computational settings for the phonon computation
+    """
+
+    # could be optional and implemented at a later stage?
+    npoints_band: int = Field("number of points for band structure computation")
+    kpath_scheme: str = Field("indicates the kpath scheme")
+    kpoint_density_dos: int = Field(
+        "number of points for computation of free energies and densities of states",
+    )
+
+
+class ThermalDisplacementData(BaseModel):
+    """
+    Collection to store information on the thermal displacement matrices
+    """
+
+    # TODO: check if we could add a connection
+    #  to a pymatgen class for thermal displacement matrices
+    freq_min_thermal_displacements: float = Field(
+        "cutoff frequency in THz to avoid numerical issues in the "
+        "computation of the thermal displacement parameters"
+    )
+    thermal_displacement_matrix_cif: List[List[Matrix3D]] = Field(
+        None, description="field including thermal displacement matrices in cif format"
+    )
+    thermal_displacement_matrix: List[List[Matrix3D]] = Field(
+        None,
+        description="field including thermal displacement matrices in cartesian coordinate system",
+    )
+    temperatures_thermal_displacements: List[int] = Field(
+        None,
+        description="temperatures at which the thermal displacement matrices"
+        "have been computed",
+    )
+
+
+class PhononUUIDs(BaseModel):
+    """
+    Collection to save all uuids connected to the phonon run
+    """
+
+    optimization_run_uuid: str = Field(None, description="optimization run uuid")
+    displacements_uuids: List[str] = Field(
+        None, description="The uuids of the displacement jobs."
+    )
+    static_run_uuid: str = Field(None, description="static run uuid")
+    born_run_uuid: str = Field(None, description="born run uuid")
+
+
+class PhononJobDirs(BaseModel):
+    """
+    Collection to save all job directories relevant for the phonon run
+    """
+
+    displacements_job_dirs: List[str] = Field(
+        None, description="The directories where the displacement jobs were run."
+    )
+    static_run_job_dir: str = Field(
+        None, description="Directory where static run was performed."
+    )
+    born_run_job_dir: str = Field(
+        None, description="Directory where born run was performed."
+    )
+    optimization_run_job_dir: str = Field(
+        None, description="Directory where optimization run was performed."
+    )
+
+
+class PhononBSDOSDoc(BaseModel):
+    """
+    Collection of all data produced by the phonon worklow
+    """
+
+    structure: Structure = Field(
+        None,
+        description="Structure of Materials Project.",
+    )
+
+    ph_bs: PhononBandStructureSymmLine = Field(
+        None,
+        description="Phonon band structure object.",
+    )
+
+    ph_dos: PhononDos = Field(
+        None,
+        description="Phonon density of states object.",
+    )
+
+    free_energy_list: List[float] = Field(
```

Comment:
You can remove the list, so it is just `free_energy` or `free_energies`

Reply:
Done